prow/test/integration/fakegitserver/.ko.yaml

```diff
@@ -0,0 +1,2 @@
+baseImageOverrides:
+  k8s.io/test-infra/prow/test/integration/fakegitserver: gcr.io/k8s-prow/git:v20220523-6026203ca9
```

Comment:
this is duplicated with the top level .ko.yaml, is it used?

Reply:
Yes. See the `Local Usage with Docker and Ko` section I wrote in the README.md.

Reply:
I see. I thought ko is smart enough to use the `.ko.yaml` file under the root, is this not the case in your local testing?

Reply:
> I thought ko is smart enough to use the .ko.yaml file under the root, is this not the case in your local testing?

Correct. Ko is smart enough but also not smart enough. If you use the default one then it errors out because it cannot find the other binaries specified in there.

Reply:
would it work if `ko` is invoked from the root of the repo?

Reply:
It appears to work if invoked from the root. I've updated the README.md to reflect this.